integrated_channels/blackboard/admin/__init__.py

```diff
@@ -78,6 +82,39 @@ def customer_oauth_authorization_url(self, obj):
         else:
             return None
 
+    def update_modified_time(self, request, obj):
+        """
+        Updates the modified time of the customer record to retransmit courses metadata
+        and redirects to configuration view with success or error message.
+        """
+        try:
+            obj.enterprise_customer.save()
+            messages.success(
+                request,
+                "The blackboard enterprise customer modified time "
+                "“<BlackboardEnterpriseCustomerConfiguration for Enterprise {enterprise_name}>” "
+                "was saved successfully.".format(
+                    enterprise_name=obj.enterprise_customer.name
+                ),
+            )
+        except ValidationError:
+            messages.error(
+                request,
+                "The blackboard enterprise customer modified time "
+                "“<BlackboardEnterpriseCustomerConfiguration for Enterprise {enterprise_name}>” "
+                "was not saved successfully.".format(
+                    enterprise_name=obj.enterprise_customer.name
+                ),
+            )
+        return HttpResponseRedirect(
+            "/admin/blackboard/blackboardenterprisecustomerconfiguration"
+        )
+    update_modified_time.label = "Update Customer Modified Time"
+    update_modified_time.short_description = (
+        "Update modified time for this Enterprise Customer "
+    )
+    "to retransmit courses metadata"
```

Comment:
This probably has to go inside the parentheses?  😂

Reply:
good catch! I used Black to auto formatter because the linters were complaining about long lines and I completely overlooked this. It's odd that this still passed all the lint and code style checks.